.changeset/beige-kings-allow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/react-brand": patch
+---
+
+[a11y] Tab action order for `FAQ Groups` according to specification
```

Comment:
Adding some verbose details for the changelog.

```suggestion
Improved keyboard navigation for tab lists in the `FAQ Groups` component. Now features focus trapping, and `up` and `down` keypress navigation as specified in the ARIA tab panel pattern.
```

Reply:
Fixed in [5c9b95c](https://github.com/primer/brand/pull/596/commits/5c9b95c462211e48f80224534a07ca0cc5de04c7)